Default.sublime-commands

```diff
@@ -162,4 +162,9 @@
         "caption": "LSP: Toggle Inlay Hints",
         "command": "lsp_toggle_inlay_hints",
     },
+    // {
+    //     "caption": "LSP: Fold All Comment Blocks",
+    //     "command": "lsp_fold_all",
+    //     "args": {"kind": "comment"}
+    // },
```

Comment:
I'm a bit undecided about it, but my intention here was to keep it in the file as an example, so it can be used as a reference in case someone asks how to add such an entry into the command palette, or maybe we decide to activate in the future. I think it would potentially be more appropriate if an LSP-* helper package would add this command, e.g. as "LSP-javascript: Fold All Comment Blocks", because they know whether the server supports the `kind` or not. In that case it would be best to create a derived command like
```python
from LSP.plugin.folding_range import LspFoldAllCommand

class LspJavascripFoldAllCommand(LspFoldAllCommand):
    session_name = 'LSP-javascript'
```
and then use that command, so that it is only active for the particular server.